plugins/mumble_plugin_win32.h

```diff
@@ -6,6 +6,8 @@
 #ifndef MUMBLE_MUMBLE_PLUGIN_WIN32_H_
 #define MUMBLE_MUMBLE_PLUGIN_WIN32_H_
 
+#include "mumble_plugin_main.h"
```

Comment:
Huh? But that file includes this file??

Reply:
I accidentally included it instead of `mumble_plugin.h`, thank you for noticing.

Reply:
I accidentally included it, thank you for noticing.